framework/src/base/Moose.C

```diff
@@ -309,6 +310,7 @@ addActionTypes(Syntax & syntax)
                            "(setup_dampers)"
                            "(setup_residual_debug)"
                            "(add_bounds_vectors)"
+                           "(add_positions)"
                            "(add_multi_app)"
```

Comment:
I assume there was a reason for this ordering. I saw at least one place (probably `MultiAppPositions`) where you had to delay data construction until `initialize` because of this ordering ... but I'm guessing the dependency issues would be worse if you flipped these